phpstan.neon

```diff
@@ -49,3 +56,17 @@ parameters:
 
         # PHPSTan false positive
         - '#Parameter \#1 \$className (.*?) expects class\-string, string given#'
+
+        -
+            message: '#Class cognitive complexity is 61, keep it under 50#'
+            path: src/NodeFinder/FormFinder.php
```

Comment:
This is sign that it should be refactor to 2 smallers classes

Reply:
Splitted to multiple classes and refactored. I've got to complexity 10 now. I can't find a way how to get to 9 :(